site/routes/definitions.js

```diff
@@ -58,12 +58,18 @@ export const routeDefinitions : Array<RouteDefinition> = [
   {
     title: ({ category }) => 'Meet our team' + (category !== 'everyone' ? ` (${category})` : ''),
     key: 'badgers',
-    route: 'about-us/people/{category?}/{page?}',
+    route: 'about-us/people+/category/{category}+/page-{page}',
     defaults: { category: 'everyone', page: 1 },
     stateToProps: stateToBadgerProps,
     gen: genBadgersParams,
   },
   {
+    title: 'Badger',
+    key: 'badger',
+    route: 'about-us/people/{name}',
+    gen: () => [],
```

Comment:
That's the route for people individual page not for listing. `gen` will get filled in when the people ind page is done